tests/test_parsers.py

```diff
@@ -154,3 +154,12 @@ def test_gelfparser_parse_with_filters(fs, gelf_logger):
     events = next(parser.parse(log_file_name, filters=[filters.anonymous]))
     assert len(events) == 1
     assert events.equals(pd.DataFrame({"username": ["foo"]}))
+
+
+def test_gelfparser_parse_server_event(fs, gelf_logger, log_generator):
+    """Test the GELFParser parsing an common 'event_sourse = server' event """
```

Comment:
This test is not required IMO as you are testing pandas JSON conversion and our GELFParser more than any logic of your fixture.